### ISSUE TITLE
document link bug

### ISSUE THREAD

**Issue by ShoufaChen** [NVIDIA/DALI#1293]
In the [ImageNet training in PyTorch](https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/examples/pytorch/pytorch-resnet50.html) docs page, the  [docs/examples/pytorch/main.py](https://github.com/NVIDIA/DALI/tree/9409b5d/docs/examples/pytorch/main.py) links to an old version of the `main.py` script, which can not run successfully.

maybe:
```
  File "main.py", line 295, in train
    input = data[0][0][0]
KeyError: 0 
```

**Comment by awolant** [comment]
Hi,
thanks for reporting this issue. I will have a fix soon.

**Comment by JanuszL** [comment]
Hi,
Docs you are referring to is from version 0.5 and it points to example valid for that version.
The most recent version that is available online is 0.13 and corresponding RN50 PyTorch doc you can find [here](https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/examples/pytorch/resnet50/pytorch-resnet50.html). It points to the right example.

**Comment by ShoufaChen** [comment]
thank you.
